docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 The latest result from a different registration formulation challenged the use of deformation regularisation and consequently the necessity of predicting dense displacement field [@hu2019conditional], also implemented in DeepReg.
 
 # Implemetation
-## Model
-A pair of moving image and fixed image is the input of the network. When available, pairs of moving labels and/or fixed labels can also be the input of the network, regardless of whether they are used to compute the loss. The network predicts a dense displacement field (DDF). Options are also available for the DDF being integrated by a predicted dense velocity field (DFV) or computed from a predicted affine transformation. For conditional segmentation, the network directly predicts warped moving labels [@hu2019conditional].
+## Registration network
+A pair of moving image and fixed image is the input of the network. When available, pairs of moving labels and/or fixed labels can also be the input of the network to compute the label dissimilarity loss. The network predicts a dense displacement field (DDF). Options are also available for the DDF being integrated by a predicted dense static velocity field, often as SVF or DVF, or computed from a predicted affine transformation. For conditional segmentation, the network directly predicts warped moving labels [@hu2019conditional].
```

Comment:
Linked with Zac's comment I think the second part also reads a bit awkward. For the whole paragraph I'd go something like this:
-->
The model takes as input a pair of fixed and a moving images and estimates a dense displacement field (DDF) using a registration network. The DDF can be directly estimated from data, integrated from a stationary velocity field (SVF or DVF) using the scaling-squaring method or computed from a predicted affine transformation. Optionally, a corresponding pair of fixed and moving labelmaps are made readily available in the objective function computation. In a conditional segmentation framework, the network directly predicts the warped moving labelmap [@hu2019conditional].